migrations/2021-12-16-003905_names_to_lowercase/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Your SQL goes here
+
+
+-- Delete person name dupes, keeping the first one
+
+delete
+from person
+where id not in (
+  select min(id)
+  from person
+  group by lower(actor_id)
+);
+
+-- Store the old values so that down.sql can recover them
+alter table person add column old_name varchar(255);
+update person set old_name = name;
+
+alter table person add column old_actor_id varchar(255);
+update person set old_actor_id = actor_id;
+
+alter table person add column old_inbox_url varchar(255);
+update person set old_inbox_url = inbox_url;
+
+
+-- Set person names, actor_id, and inbox_url to lowercase
+update person set name = lower(name);
+update person set actor_id = lower(actor_id);
+update person set inbox_url = lower(inbox_url);
+
+
+
+-- Add a lowecase enforcement check to these three columns
+
+alter table person add constraint person_name_lowercase_ck check (name = lower(name));
+alter table person add constraint person_actor_id_lowercase_ck check (actor_id = lower(actor_id));
```

Comment:
I think there's a idx_group `lower` index, but you should probably remove that old one, as this new one is better.

Reply:
Currently, there is a lower_actor_id unique index constraint. 

Indexes:
    "person__pkey" PRIMARY KEY, btree (id)
    "idx_person_actor_id" UNIQUE CONSTRAINT, btree (actor_id)
    "idx_person_inbox_url" UNIQUE CONSTRAINT, btree (inbox_url)
    "idx_person_lower_actor_id" UNIQUE, btree (lower(actor_id::text))
    "idx_person_published" btree (published DESC)